api/v1alpha/framework/testcafe.schema.json

```diff
@@ -51,6 +51,7 @@
           "$ref": "../subschema/common.schema.json#/definitions/version",
           "enum": [
             "package.json",
```

Comment:
Don't think `1.20.0` is supported any longer. We should remove that as well.

Reply:
Forgot about it. Just removed.